README.md

```diff
@@ -5,9 +5,9 @@ This includes shared libraries, used by SDKs and other tools, as well as SDKs.
 
 ## Packages
 
-| Readme                                       | issues                                      | tests                                                   | docs                                                                                                                                                                           |
-|----------------------------------------------|---------------------------------------------|---------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [libs/client-sdk](libs/client-sdk/README.md) | [C++ Client SDK][package-cpp-client-issues] | [![Actions Status][cpp-client-ci-badge]][cpp-client-ci] | [![Documentation](https://img.shields.io/static/v1?label=GitHub+Pages&message=API+reference&color=00add8)](https://launchdarkly.github.io/cpp-sdks/libs/client-sdk/docs/html/) |
+| Readme                                       | issues                                      | tests                                                   | docs                                                                                                                                                                           | latest release                                                                           |
+|----------------------------------------------|---------------------------------------------|---------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| [libs/client-sdk](libs/client-sdk/README.md) | [C++ Client SDK][package-cpp-client-issues] | [![Actions Status][cpp-client-ci-badge]][cpp-client-ci] | [![Documentation](https://img.shields.io/static/v1?label=GitHub+Pages&message=API+reference&color=00add8)](https://launchdarkly.github.io/cpp-sdks/libs/client-sdk/docs/html/) | [On Github](https://github.com/launchdarkly/cpp-sdks/releases?q=launchdarkly-cpp-client) | 
```

Comment:
I would use this URL instead:
`https://github.com/launchdarkly/cpp-sdks/releases?q=%22launchdarkly-cpp-client%22`

It will only show the client, without the quotes you also get the SSE client.